homework3.php

```diff
@@ -0,0 +1,21 @@
+<?php
+//homework3
+function checkPrimeNumber($n) {
+    if($n == 1){
+      echo"1 isn't Prime Number";
+    }else{
```

Comment:
You should handle special case first, and return the result for this special case. You used many nested `if-else`, your source code will be not clear.

Please check hint at https://gitlab.com/pascaliaasia/pav-training/-/blob/basic-training/basic-programing/php/homework-hint/php-hint.md